ssg/build_remediations.py

```diff
@@ -346,8 +346,7 @@ def _get_rule_reference(self, ref_class):
     def update_when_from_rule(self, to_update):
         additional_when = ""
         if self.associated_rule.platform == "machine":
-            additional_when = ('ansible_virtualization_role != "guest" '
-                               'or ansible_virtualization_type != "docker"')
+            additional_when = 'ansible_virtualization_type not in ["docker", "lxc", "openvz"]'
```

Comment:
How about `podman` and `container` types?

Reply:
@yuumasato current versions of Ansible don't know what Ansible is, so I didn't want to mislead anyone in thinking that we are somehow checking for it. Make sense? This won't be available until 2.10 stable release. I'll create an issue to track that when stable 2.10 is available, we can make the changes.

Reply:
Makes sense, for the `podman` and `container` types.

Ping  @matejak for awareness of the change in `ansible_virtualization_role`.